courses/javascript/classes.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Introduction To Javascript"
+subheading: "Class"
+prev: "arrow-functions"
+next: ""
+testCase: [
+    {
+      id: 1,
+      case: ["class Calcu {"],
+      hint: "Please re check the class syntax and code formatting.",
+      isCorrect: false
+    },
+     {
+      id: 2,
+      case: ["constructor(num1){"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+     {
+      id: 3,
+      case: ["this.num1 = num1;"],
+      hint: "Please re check the csode and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 4,
+      case: ["}"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 5,
+      case: ["toAdd(num2) {"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 6,
+      case: ["return this.num1 + num2;"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 7,
+      case: ["}"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 8,
+      case: ["}"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 9,
+      case: ["let calcuObjects = new Calcu(1);"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+     {
+      id: 10,
+      case: ["const calcuNum2 = calcuObjects.toAdd(2);"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+    {
+      id: 11,
+      case: ["console.log(concatData);"],
+      hint: "Please re check the code and formatting.",
+      isCorrect: false
+    },
+]
+---
+
+A javascript class is a blueprint for creating objects. A class encapsulates data and functions that manipulate data.
```

Comment:
class wrapped in backtick(``) which makes it inline code block. All technical terms need to be wrapped in inline code block